package.json

```diff
@@ -124,23 +124,40 @@
 			},
 			{
 				"command": "sourcery.rule.create",
-				"title": "Create a custom rule for Sourcery",
+				"title": "Create custom rule",
 				"category": "Sourcery"
 			}
 		],
 		"submenus": [
 			{
 				"id": "sourcery.scans",
 				"label": "Sourcery"
+			},
+			{
+				"id": "sourcery.editor",
+				"label": "Sourcery"
 			}
 		],
 		"menus": {
+			"editor/context": [
+				{
+					"submenu": "sourcery.editor",
+					"group": "z_commands@2"
```

Comment:
I have no clue about what this does... If this file is manually edited (I guess it is), could you please provide a comment here just in case someone needs to change this behavior in the future? Googling "group z_commands@2" is of no help :sweat_smile:

Reply:
Can't add comments to the json file, but good point - not sure where to put notes like this. This stuff is all in the VS Code extension API documentation.